express_relay/testing/src/express_relay/set_router_split.rs

```diff
@@ -0,0 +1,32 @@
+use anchor_lang::{InstructionData, ToAccountMetas};
+use solana_sdk::{instruction::Instruction, pubkey::Pubkey, signature::Keypair, signer::Signer, system_program};
+use express_relay::{accounts::SetRouterSplit, SetRouterSplitArgs};
+
+use super::helpers::{get_express_relay_metadata_key, get_router_config_key};
+
+pub fn get_set_router_split_instruction(
```

Comment:
get_set is a bit confusing. It has a different meaning in some [contexts](https://redis.io/docs/latest/commands/getset/).  maybe `make_set` or `create_set`?